src/main/java/ua/biz/test/config/Application.java

```diff
@@ -0,0 +1,88 @@
+package ua.biz.test.config;
+
+import ua.biz.test.data.DBCredential;
+import ua.biz.test.data.TableAccess;
+import ua.biz.test.pojo.Employee;
+import ua.biz.test.pojo.JobVacancy;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Application {
+    private List<Employee> employees;
+
+    public Application() {
+        employees = new ArrayList<>();
+    }
+
+    public void getEmployee() {
```

Comment:
get with VOID is a big punch for conventions

Reply:
fixed